docs/serving/configuration/deployment.md

```diff
@@ -106,3 +109,27 @@ data:
   # List of repositories for which tag to digest resolving should be skipped
   registries-skipping-tag-resolving: registry.example.com
 ```
+
+## Enable Freezer Service
+
+You can configure queue-proxy to call a endpoint when it's traffic drops to zero or scales up from zero by modifying `concurrency-state-endpoint` ConfigMap setting. With this configuration, you can achieve some features like freezing runing process in user-contianer or charing calculation based on time of handling requests。
+
+Before you configure this, your need to implement this endpoint API. The official implementation is [container-freezer](https://github.com/knative-sandbox/container-freezer).
+
+The following example show how to enable Freezer Service, $HOST_IP is node's IP:
```

Comment:
We actually want users to use `$HOST_IP` (and not a specific host's IP) here -- the freezer will swap in the appropriate value for each node at runtime

Reply:
fixde